hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -87,6 +135,12 @@ class HassioIngressView extends LitElement {
         height: 100%;
         border: 0;
       }
+
+      hass-subpage.config {
+        --app-header-background-color: var(--sidebar-background-color);
```

Comment:
Seems weird that we use the sidebar background for this? I think that it is better suited to stay primary color, as that makes it blend into the phone header, which is also styled like the theme.

Reply:
This is just for when it is in the config panel, so it matches the color of the config header